src/index.html

```diff
@@ -17,61 +17,65 @@
     <div class="container text-center">
       <div class="row row-cols-2 row-cols-lg-5 g-2 g-lg-3">
         <div class="col">
-          <div class="p-3" type="button" class="collapse-btn" onclick="toggleCollapse('collapseContent1')">Weather</div>
-           <div class="collapse-content" id="collapseContent1" style="display: none">
-           <p>This is the collapsible content that will be shown or hidden.</p>
-           </div>
+          <div class="p-3 d-grid gap-2">
+            <button class="btn btn-primary btn-lg" type="button" data-bs-toggle="collapse" data-bs-target="#collapseContent1" aria-expanded="false" aria-controls="collapseContent1">
+              Weather
+            </button>
+            <div class="collapse collapse-content" id="collapseContent1">
+              <p>This is the collapsible content that will be shown or hidden.</p>
```

Comment:
This element is your content that will be shown/hidden by the collapse.
We don't need to apply an explicit `style="display: none"`, because the class `collapse` does this for us. 
This class will ensure it's hidden by default.

The only other thing you need here is `id="collapseContent1"` should match what you add to `data-bs-target` in the corresponding button.

Reply:
You may also notice I've wrapped both the button, and the collapse inside the same `div`.
This isn't mandatory, they'll work no matter where in the DOM they are.

I've only done this so that we apply the same spacing/styling to everything in that div `class="p-3 d-grid gap-2"`.
`p-3` means padding 3
`d-grid` and `gap-2` I stole from [Block buttons](https://getbootstrap.com/docs/5.3/components/buttons/#block-buttons) in the bootstrap docs. This makes the button take up the full column, like you did with your custom buttons.

If we don't do this, your collapse content may spill out further than your button, and could look a bit weird. As I say, not critical.